subschemas/substrate-chain/src/resolvers/Query/councilMotions.ts

```diff
@@ -23,21 +25,32 @@ export async function councilMotions(
 
   return await Promise.all(
     motions.map(async (motion) => {
-      const proposal = {
-        hash: String(motion.proposal.hash),
-        ...getCallParams(motion.proposal),
-      };
-
-      return {
-        hash: String(motion.hash),
-        proposal,
-        votes: motion.votes ? await getVotes(motion.votes, accountService, api) : undefined,
-        votingStatus: motion.votes ? getVotingStatus(motion.votes, councilMembers.length, bestNumber, api) : undefined,
-      };
+      return (await getMotionDetails(motion, accountService, api, councilMembers, bestNumber)) as CouncilMotion;
```

Comment:
There is no need to async/await here, you are already doing `Promise.all`. Why do you need to do `as CouncilMotion`?

Reply:
Yeah, right, it's not ok - I'll change it - it was the result of a refactoring that I didn't cleanup properly 👍